### PR TITLE
chore: update cypress tests to vitest

### PR DESCRIPTION
**Pull request by iNSaNiA0821**
- [x] composables/display
- [x] composables/display-components
- [x] composables/goto
- [x] composables/resizeObserver
- [x] composables/scroll
- [x] directives/intersect
- [x] directives/scroll
- [x] directives/touch
- [x] framework

Tests with percy snapshots:

- [x] VAlert
- [x] VAutocomplete
- [x] VBadge
- [x] VBtn
- [x] VCombobox
- [x] VFileInput
- [x] VInput
- [x] VSelect
- [x] VSwitch
- [x] VTextField
- [x] VVirtualScroll

Everything else:

- [x] VAppBar
- [x] VBottomNavigation
- [x] VBottomSheet
- [x] VBreadcrumbs
- [x] VBtnGroup
- [x] VCheckboxBtn
- [x] VChip
- [x] VColorPicker
- [x] VConfirmEdit
- [x] VDataIterator
- [x] VDataTable
- [x] VDataTableServer
- [x] VDataTableVirtual
- [ ] VDatePicker
- [ ] VDefaultsProvider
- [ ] VDialog
- [x] VDivider
- [ ] VExpansionPanels
- [ ] VForm
- [ ] VHover
- [x] VIcon
- [ ] VInfiniteScroll
- [ ] VItemGroup
- [ ] VLayout
- [ ] VList
- [ ] VListGroup
- [x] VListItem
- [x] VMain
- [ ] VNavigationDrawer
- [x] VNumberInput
- [ ] VOtpInput
- [ ] VOverlay
- [ ] VPagination
- [ ] VProgressLinear
- [ ] VRating
- [x] VSelectionControl
- [ ] VSlideGroup
- [ ] VSlider
- [ ] VSystemBar
- [ ] VTabs
- [ ] VTextarea
- [ ] VThemeProvider
- [ ] VTimeline
- [ ] VToolbar
- [x] VTreeview
- [ ] VWindow

### CODE REVIEW

**Comment by KaelWD** [comment]
@iNSaNiA0821 please use the functions from `@test` where possible, see c315b4fbcc193432713218b7db212ce92ec526c0

- prefer ref() over rerender()
- use expect.element or expect.poll instead of nextTick
- use screen instead of querySelector

**Comment by iNSaNiA0821** [comment]
@KaelWD Thanks for the feedback! I’ll update the code accordingly.

**Comment by iNSaNiA0821** [comment]
@KaelWD, Wishing you a wonderful holiday!
I’ve refactored the Vitest files as per your guidance. Could you please review them?

File: packages/vuetify/src/components/VDataTable/__tests__/VDataTable.spec.browser.tsx [hunk 1]
```diff
@@ -1,107 +0,0 @@
-// Components
-import { VDataTable } from '../'
-
-// Utilities
-import { render, screen, userEvent } from '@test'
-import { ref } from 'vue'
-
-describe('VDataTable', () => {
-  it('mirrors external updates', async () => {
```
Comment (KaelWD) [inline packages/vuetify/src/components/VDataTable/__tests__/VDataTable.spec.browser.tsx]: These new tests you added don't actually check anything that is likely to ever break and there's lots of tests missing from the originals. I undid these changes so I could merge it, they're back in the branch now though.

**Pull request merged**